common/icons/chevron.js

```diff
@@ -1,5 +1,3 @@
-const IconChevron = () => (
-  <svg viewBox='0 0 26 26'><path d='M18.8 10.45a1 1 0 0 0-1.41 0L13 14.87l-4.44-4.42a1 1 0 1 0-1.41 1.41L12.27 17a1 1 0 0 0 1.41 0l5.12-5.12a1 1 0 0 0 0-1.43z'/></svg>
+export default () => (
+  <svg viewBox="0 0 26 26"><path d="M18.8 10.45a1 1 0 0 0-1.41 0L13 14.87l-4.44-4.42a1 1 0 1 0-1.41 1.41L12.27 17a1 1 0 0 0 1.41 0l5.12-5.12a1 1 0 0 0 0-1.43z"/></svg>
```

Comment:
Are double-quotes for svg important?

Reply:
Copy paste, so don't suppose it really matters - if we start caring about this, let's automate 🤖